docs/reference/intro.asciidoc

```diff
@@ -146,90 +170,197 @@ A mapping defines the <<mapping-types,data type>> for each field, how the field
 and how it should be stored.
 When adding documents to {es}, you have two options for mappings:
 
-* <<mapping-dynamic, Dynamic mapping>>: Let {es} automatically detect the data types and create the mappings for you. This is great for getting started quickly, but can lead to unexpected results for complex data.
-* <<mapping-explicit, Explicit mapping>>: Define the mappings up front by specifying data types for each field. Recommended for production use cases, because you have much more control over how your data is indexed.
+* <<mapping-dynamic, Dynamic mapping>>: Let {es} automatically detect the data types and create the mappings for you. This is great for getting started quickly, but might yield suboptimal results for your specific use case due to automatic field type inference.
```

Comment:
:2c: avoid `this is great for` due to vagueness

```suggestion
* <<mapping-dynamic, Dynamic mapping>>: Let {es} automatically detect the data types and create the mappings for you. Dynamic mapping is useful for getting started quickly, but might yield suboptimal results for your specific use case due to automatic field type inference.
```

Reply:
I would contend that "useful" is equally vague but we're getting into the nitty weeds

Maybe we can go from "Great to good"? Still vague but feels nicer in active voice?